### DISCUSSION TITLE
updateUri bug when change site language with naviagate

### DISCUSSION THREAD

**Discussion by h-avatar** [livewire/livewire#6068] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Hi, 
I write site language component with navigate feature and put short language name in url. 
But after i change language livewire make request to old language url, I check livewire.js and found out it loaded once on page load. thus updateUri variable is not updated.
I fix this issues with put this variable inside sendRequestToServer function. is it correct way?

**Comment by joshhanley** [comment]
@h-avatar sorry I don't follow what the issue is. Can you please provide some code and screenshots to demonstrate the issue? Thanks!

**Reply by h-avatar** [reply]
Sorry, my English skill is not good.
I wanted to change the language of the page without refreshing the page using the navigate feature. I was able to do this, but the address that Livewire was requesting was the old language address. Therefore, I transferred the updateUri variable in the livewire.js file to the sendRequestToServer function and my problem was solved.
With this method, every time Livewire makes a request, it receives a new address.

![livewire](https://github.com/livewire/livewire/assets/29028910/277346ff-a8c7-4c4d-8102-133dcd745aa4)

**Reply by joshhanley** [reply]
@h-avatar ah ok. Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it? Thanks!
